From 193a4cb7f63f5bbf0af0379d723e415406f3acd1 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Fri, 8 Feb 2013 09:55:53 -0500
Subject: [PATCH] Revert "Do not show the titlebar when the window is
 maximized"

This reverts commit 6b79b0085b6c8630ea21b27be6f35ccec99784e2.

Hiding the titlebar really only works in applications that have
been designed with this pattern in mind. In particular, it does
not work with a menubar, and really requires a draggable toolbar.

See https://bugzilla.gnome.org/show_bug.cgi?id=692580 for further
discussion.
---
 src/terminal-window.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/terminal-window.c b/src/terminal-window.c
index 183fba7..8162132 100644
--- a/src/terminal-window.c
+++ b/src/terminal-window.c
@@ -1822,10 +1822,8 @@ terminal_window_init (TerminalWindow *window)
 #endif
 
   gtk_window_set_title (GTK_WINDOW (window), _("Terminal"));
-  gtk_window_set_hide_titlebar_when_maximized (GTK_WINDOW (window), TRUE);
 
   priv->active_screen = NULL;
-  priv->menubar_visible = FALSE;
 
   main_vbox = gtk_box_new (GTK_ORIENTATION_VERTICAL, 0);
   gtk_container_add (GTK_CONTAINER (window), main_vbox);
-- 
1.8.1.4

